cmd/oapi-codegen/oapi-codegen.go

```diff
@@ -265,6 +268,27 @@ func updateConfigFromFlags(cfg configuration) (configuration, error) {
 	return cfg, nil
 }
 
+// updateTemplatesFromFiles populates the UserTemplates field in OutputOptions using the
+// UserTemplateFiles field. If there is a shared key between UserTemplates and UserTemplateFiles,
+// the contents of the file take precedence.
+func updateTemplatesFromFiles(cfg configuration) (configuration, error) {
+	oo := &cfg.OutputOptions
+	if oo.UserTemplateFiles == nil {
+		return cfg, nil
+	}
+	if oo.UserTemplates == nil {
+		oo.UserTemplates = make(map[string]string)
+	}
+	for key, filename := range oo.UserTemplateFiles {
+		data, err := ioutil.ReadFile(filename)
+		if err != nil {
+			return configuration{}, err
```

Comment:
Wrap the error to provide the needed context to the user to solve the issue.
```suggestion
			return fmt.Error("load user templates %q: %w", key, err)
```